vendor/assets/javascripts/selectize.js

```diff
@@ -2652,7 +2652,8 @@
 			var invalid, self = this;
 			if (self.isRequired) {
 				if (self.items.length) self.isInvalid = false;
-				self.$control_input.prop('required', invalid);
+				self.$control_input.prop('required', self.$input.val() == '');
+				self.$input.prop('required', false);
```

Comment:
Is this upgrading Selectize, or a manual tweak? Would strongly suggest avoiding modifying 3rd-party libraries, and if we do, put a note at the top with the corresponding pull request to the original, or at least explaining the change.